test/BinaryToText.cpp

```diff
@@ -245,7 +245,10 @@ using RoundTripInstructionsTest =
     spvtest::TextToBinaryTestBase<::testing::TestWithParam<std::string>>;
 
 TEST_P(RoundTripInstructionsTest, Sample) {
-  EXPECT_THAT(EncodeAndDecodeSuccessfully(GetParam()), Eq(GetParam()));
+  EXPECT_THAT(
+      EncodeAndDecodeSuccessfully(GetParam(), SPV_BINARY_TO_TEXT_OPTION_NONE,
+                                  SPV_ENV_UNIVERSAL_1_1),
```

Comment:
Fixed.
New commit parameterizes the BinaryToText round trip tests over both v1.0 and 1.1.
The new dependency operands to OpLoopMerge are only tested in 1.1.